<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersWidget</class>
 <widget class="QWidget" name="UsersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayoutUsers" columnstretch="6,4">
   <item row="0" column="0">
    <widget class="QListWidget" name="userList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEditUsername">
       <property name="placeholderText">
        <string>Username</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPasswordConfirm">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Confirm Password</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addUserButton">
       <property name="text">
        <string>Add User</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
